installer.sh

```diff
@@ -605,7 +605,7 @@ echo "==========================================================================
 if [ ! -d "/var/lib/keylime/tpm_cert_store" ]; then
   echo "Creating new tpm_cert_store"
   mkdir -p /var/lib/keylime
-  mv $KEYLIME_DIR/tpm_cert_store /var/lib/keylime/tpm_cert_store
+  cp $KEYLIME_DIR/tpm_cert_store /var/lib/keylime/tpm_cert_store
```

Comment:
This should be `cp -r`.

Reply:
@kgold2 can you update this PR with the `-r` option to `cp`?

Reply:
Actually this has been fixed in another PR. @kgold2 can you rebase this commit off of master so your specific fixes can be applied?

Reply:
```
git fetch --all
git rebase origin/master
```